src/MissionManager/SurveyMissionItem.cc

```diff
@@ -1062,7 +1062,7 @@ bool SurveyMissionItem::_appendMissionItemsWorker(QList<MissionItem*>& items, QO
         MissionItem* item = new MissionItem(seqNum++,
```

Comment:
Ah, it's `_imagesEverywhere`. Since it is not set, we never reach execute the inner statement. I'm not sure why we need to check that at the end of the survey anyway. Can we get rid of the check?

Also, related - my open concern No. 1 in the PR's opening post.